README.md

```diff
@@ -0,0 +1,101 @@
+# Tekton Task Catalog
```

Comment:
I was thinking this would eventually contain both tasks and pipelines. WDYT?

Reply:
Yeah definitely. I only called this the "Task Catalog" because it only has Tasks for right now.

Reply:
and resources one day too (once we decouple them from the controller... WATCH THIS SPACE...)